reconcile/terraform_resources.py

```diff
@@ -115,18 +117,31 @@
 def populate_oc_resources(spec, ri):
     if spec.oc is None:
         return
-    for item in spec.oc.get_items(spec.resource,
-                                  namespace=spec.namespace):
-        openshift_resource = OR(item,
-                                QONTRACT_INTEGRATION,
-                                QONTRACT_INTEGRATION_VERSION)
-        ri.add_current(
-            spec.cluster,
-            spec.namespace,
-            spec.resource,
-            openshift_resource.name,
-            openshift_resource
-        )
+
+    logging.debug("[populate_oc_resources] cluster: "+ spec.cluster \
+                + " namespace: "+ spec.namespace \
+                + " resource: "+ spec.resource)
+
+    try:
+      for item in spec.oc.get_items(spec.resource,
+                                    namespace=spec.namespace):
+          openshift_resource = OR(item,
+                                  QONTRACT_INTEGRATION,
+                                  QONTRACT_INTEGRATION_VERSION)
+          ri.add_current(
+              spec.cluster,
+              spec.namespace,
+              spec.resource,
+              openshift_resource.name,
+              openshift_resource
+          )
+    except StatusCodeError as e:
+      logging.error("[populate_oc_resources] :"+str(e))
```

Comment:
I would say we don't care about what the error is.  If there's an error, just print it and the cluster it is associated with.  Looking fo specific string in an error message is inherently fragile, and in this case I really don't think we care.  We just want to know what cluster had an error, right?

Reply:
We may want to handle errors depending on what they are. Some may require us to stop processing and other may allow us to continue